optimas/explorations/base.py

```diff
@@ -124,6 +124,12 @@ def __init__(
         self._set_default_libe_specs()
         self._libe_history = self._create_libe_history()
         self._load_history(history, resume)
+        self._is_manager = self._set_manager(self.libe_comms, self.libE_specs)
+
+    @property
+    def is_manager(self):
+        """Get whether the current process is the manager"""
```

Comment:
```suggestion
        """Get whether the current process is the manager."""
```
Add a dot in the end to appease the CI bot.